stable/yugaware/values.yaml

```diff
@@ -62,8 +62,12 @@ yugaware:
       cpu: 2
       memory: 4Gi
   enableProxyMetricsAuth: true
+  ## List of additional alowed CORS origins in case of complex rev-proxy
+  additionAllowedCorsOrigins: []
+  #   - http://external.host.domain.local
+  #   - https://mylocalhost
```

Comment:
I think these comments are a bit misleading. If the user uncomments these lines, the above empty list will get in the way of being able to use them. Would propose to remove these comments. Otherwise, lgtm.

Reply:
NP, will remove comments